paasta_tools/check_chronos_jobs.py

```diff
@@ -90,6 +90,23 @@ def sensu_event_for_last_run_state(state):
         return pysensu_yelp.Status.OK
 
 
+def merge_duplicate_jobs(jobs):
```

Comment:
Is this really the right name for what this does?

Doesn't this function actually find_most_recent_jobs ?